doc/misc/system-state.7.rst

```diff
@@ -0,0 +1,44 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _systemstate_misc_ref-label:
+
+##############
+ System state
+##############
+
+Description
+===========
+
+The DNF5 system state consists of several TOML files located in the /usr/lib/sysimage/libdnf5/ directory. DNF5 uses the system state to:
```

Comment:
The `/usr/lib/sysimage/libdnf5/` is the default but I think we should mention that the location is taken from `system_state_dir` config option.

Reply:
Right. I'll fix it.

Reply:
Done.